src/community/blocks.tsx

```diff
@@ -0,0 +1,277 @@
+    return (
+        <div aria-label={ariaLabel}>
+            <Inline space={16} alignItems="center">
+                {image}
+                <Text2 regular color={vars.colors.textSecondary}>
+                    {description}
+                </Text2>
+            </Inline>
+        </div>
+    );
+};
+
+interface InformationBlockProps {
+    title?: string;
+    description?: Array<string>;
+    value?: number;
+    secondaryValue?: number;
+    valueColor?: string;
+    'aria-label'?: string;
+}
+
+export const InformationBlock: React.FC<InformationBlockProps> = ({
+    title,
+    description,
+    secondaryValue,
+    value,
+    valueColor = vars.colors.textPrimary,
+    'aria-label': ariaLabel,
+}) => {
+    return (
+        <Inline space="between" alignItems="flex-end" aria-label={ariaLabel}>
+            <BlockContent title={title} description={description} />
+            <div className={classNames(styles.column, styles.rightContent)}>
+                <Text2 regular color={vars.colors.textSecondary}>
+                    {secondaryValue}
+                </Text2>
+                <Text5 color={valueColor}>{value}</Text5>
+            </div>
+        </Inline>
+    );
+};
+
+interface HighlightedValueBlockProps {
+    headline?: RendersNullableElement<typeof Tag>;
+
+    mainHeading: {
+        value: number;
+        text: string;
+    };
+
+    secondHeading?: {
+        value: number;
+        text: string;
+    };
+
+    secondaryValue?: number;
+
+    title?: string;
+    description?: Array<string>;
+
+    valueColor?: string;
+    'aria-label'?: string;
+}
+
+export const HighlightedValueBlock: React.FC<HighlightedValueBlockProps> = ({
+    headline,
+    mainHeading,
+    secondHeading,
+    secondaryValue,
+    title,
+    description,
+    valueColor = vars.colors.textPrimary,
+    'aria-label': ariaLabel,
+}) => {
+    return (
+        <div aria-label={ariaLabel}>
+            {headline && <Box paddingBottom={24}>{headline}</Box>}
+
+            <Stack space={2}>
+                <Inline space={8} alignItems="baseline">
+                    <Text8 color={valueColor}>{mainHeading.value}</Text8>
+                    <Text2 regular color={vars.colors.textSecondary}>
+                        {mainHeading.text}
+                    </Text2>
+                </Inline>
+                {secondHeading && (
+                    <Inline space={8} alignItems="baseline">
+                        <Text8 color={valueColor}>{secondHeading.value}</Text8>
+                        <Text2 regular color={vars.colors.textSecondary}>
+                            {secondHeading.text}
+                        </Text2>
+                    </Inline>
+                )}
+                {secondaryValue && <Text8 color={vars.colors.textSecondary}>{secondaryValue}</Text8>}
+            </Stack>
+            <Box paddingTop={8}>
+                <BlockContent title={title} description={description} />
+            </Box>
```

Comment:
I'd render the Box conditionally. If there isn't title/description you don't need an extra padding:
```suggestion
            {title || description ? (
                <Box paddingTop={8}>
                    <BlockContent title={title} description={description} />
                </Box>
            ) : null}
```